src/command/helpModule.ts

```diff
@@ -40,6 +41,34 @@ export default class HelpModule extends Module {
             );
         }, initialEmbed);
 
-        await msg.channel.send(embed);
+        const fallbackContent = modules.reduce((message, module) => {
+            const commands = _commands.filter(
+                command => command.module === module
+            );
+            return message.concat(
+                module.constructor.name,
```

Comment:
Perhaps use markdown in the code block and make this `# ${module.constructor.name}`?
P.S: You'll need to get rid of the [extra newline](https://i.ronthecookie.me/N0RhARs.png).

Reply:
Sure thing!